torch/csrc/jit/passes/onnx/initializer_based_peephole.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <torch/csrc/jit/ir/ir.h>
+
+namespace torch {
+namespace jit {
+
+void InitializerPeepholeONNX(
```

Comment:
Similarly, rename this function and file to reflect that this is an ONNX pass for eval/inference peephole optimization.